daml-assistant/exe/DA/Daml/Assistant.hs

```diff
@@ -302,7 +302,7 @@ withLogger damlPath k = do
     let logOfInterest prio = prio `elem` [L.Telemetry, L.Warning, L.Error]
         gcpConfig = L.GCPConfig
             { gcpConfigTag = "assistant"
```

Comment:
this seems off, we don’t set the cache path at all and we use `damlPath` instead. Telemetry in the assistant shouldn’t behave different from telemetry in damlc.

Reply:
By setting it to `Nothing`, we will try to use the XDG_CACHE_DIR and use this when set.

Reply:
Ah right but the `optedOutPath` and `optedInPath` below still need to take the actual cache directory into account instead of being relative to `damlPath`.

Reply:
ah, that's true. Good catch!